src/systems/camera.js

```diff
@@ -20,24 +27,53 @@ module.exports.System = registerSystem('camera', {
    * entities at the origin (0, 0, 0) are well-centered.
    */
   setupDefaultCamera: function () {
+    var self = this;
     var sceneEl = this.sceneEl;
     var defaultCameraEl;
     // setTimeout in case the camera is being set dynamically with a setAttribute.
-    setTimeout(function checkForCamera () {
+    setTimeout(checkForCamera);
+    function checkForCamera () {
       var currentCamera = sceneEl.camera;
       if (currentCamera) {
         sceneEl.emit('camera-ready', { cameraEl: currentCamera.el });
         return;
       }
-
       defaultCameraEl = document.createElement('a-entity');
-      defaultCameraEl.setAttribute('position', {x: 0, y: 1.8, z: 4});
+      defaultCameraEl.setAttribute('position', DEFAULT_CAMERA_POSITION);
       defaultCameraEl.setAttribute(DEFAULT_CAMERA_ATTR, '');
       defaultCameraEl.setAttribute('camera', {'active': true});
       defaultCameraEl.setAttribute('wasd-controls', '');
       defaultCameraEl.setAttribute('look-controls', '');
       sceneEl.appendChild(defaultCameraEl);
+      sceneEl.addEventListener('enter-vr', self.removeDefaultOffset);
+      sceneEl.addEventListener('exit-vr', self.addDefaultOffset);
       sceneEl.emit('camera-ready', {cameraEl: defaultCameraEl});
+    }
+  },
+
+  addDefaultOffset: function () {

```

Comment:
add/remove can use docstrings, and the `Remove default camera if present` comments can be removed